src/Microsoft.Health.Dicom.Core.UnitTests/Extensions/ExtendedQueryTagEntryExtensionsTests.cs

```diff
@@ -36,6 +36,15 @@ public void GivenPrivateTagWithNonEmptyPrivateCreator_WhenNormalizing_ThenPrivat
             Assert.Null(normalized.PrivateCreator);
         }
 
+        [Fact]
+
+        public void GivenPrivateIdentificationCodeTagWithoutVR_WhenNormalizing_ThenVRShouldBeFilled()
+        {
+            ExtendedQueryTagEntry entry = CreateExtendedQueryTagEntry("22010010", null, null, QueryTagLevel.Instance, ExtendedQueryTagStatus.Ready);
+            ExtendedQueryTagEntry normalized = entry.Normalize(ExtendedQueryTagStatus.Adding);
+            Assert.Equal(DicomVRCode.LO, normalized.VR);
```

Comment:
Should there be a test case for when the VR is lowercase?

Reply:
yes, https://github.com/microsoft/dicom-server/blob/db1e430e47a3084b4a9b8d1fd4701caf914ed2d5/src/Microsoft.Health.Dicom.Core.UnitTests/Extensions/ExtendedQueryTagEntryExtensionsTests.cs#L71 covers that